### PR TITLE
Dropdown Component

### PR DESCRIPTION
**Pull request by aconchatorres**
### Description
The component equivalent to the select /option tags is created and implements the following properties:

* __`options`:__ Array prop that contains the options that will be available to select from the dropdown. Each option must have a `value` and a `text`.
  ##### Example:
      options: [
        { value: null, text: 'Please select an option' },
        { value: 'a', text: 'This is First option' },
        { value: 'b', text: 'Selected Option' },
        { value: { C: '3PO' }, text: 'This is an option with object value' },
        { value: 'd', text: 'This one is disabled', disabled: true }
      ]

_It is important to create the first value with the hint text and a `null` value. This option will be displayed as default when the component is created._

  #### Handle onChangeValue:
To run a function when a different option is picked, it can be used the Vue `watch` property to listen to the changed variable that is assinged in the `v-model`.
  ##### Example:
    data() {
      return {
        variable_name: ''
      }
    },
    watch: {
      variable_name: function (val) {
      // when the variable_name prop changes, this function will be fired.
      function_name()
    }

### CODE REVIEW

File: components/global/AppDropdown.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.global {
+  background: $t-black-5;
+  width: 100%;
+  color: black;
```
Comment (jfarellano) [inline components/global/AppDropdown.vue]: $t-balack instead of black

File: components/global/AppDropdown.vue [hunk 2]
```diff
@@ -0,0 +1,47 @@
+<template>
```
Comment (jfarellano) [inline components/global/AppDropdown.vue]: pug

File: components/global/AppDropdown.vue (hunk 1 above)
Comment (aconchatorres) [reply components/global/AppDropdown.vue]: `color: black;` changed to `color: $t-black;`

File: components/global/AppDropdown.vue (hunk 2 above)
Comment (aconchatorres) [reply components/global/AppDropdown.vue]: Component template changed to PUG

**Pull request merged**